domains/harshyadav.json

```diff
@@ -0,0 +1,8 @@
+{
+    "owner": {
+        "username": "iharshyadav"
+    },
+    "records": {
+        "A": "216.198.79.1"
```

Comment:
Your A records are not properly done. A records should be presented in an array of strings as opposed to a single string.

Reply:
Re-open this pull request when you have that fixed up.